openpnm/utils/Project.py

```diff
@@ -579,28 +581,40 @@ def export_data(self, phases=[], filename=None, filetype='vtp'):
         network = self.network
         if filename is None:
             filename = project.name + '_' + time.strftime('%Y%b%d_%H%M%p')
-        # Infer filetype from extension on file name...if given
+        # Check if any of the phases has time series
```

Comment:
I don't like checking this here.  I think this should be checked within each specific io class, or better yet, should be added to the GenericIO class so that all classes get it.  For instance, add a method called ``is_transient`` and they add a check to all classes that raise an exception if it returns ``True``.  It's a bit more work, but more general and maintainable going forward.  Also, the ``phases`` argument should accept a single object or a list of objects, so be sure to check for that.

Reply:
@jgostick Thanks for reviewing. I agree, it's better.
You want to raise an exception if transient data are present and the user wants to export in format other than XDMF? Isn't raising a warning and exporting in XDMF format better?

Reply:
no, raise an exception, with a message to use xdmf.  You're way is more convenient for sure, but it's not a good idea to write a bunch of files that the user didn't ask for.